RefData/data/10051_add_service_ids_and_jurisidictions.sql

```diff
@@ -0,0 +1,60 @@
+SET XACT_ABORT ON
+GO
+
+CREATE OR ALTER PROC #Jurisdiction_CreateIfNotExist @name varchar(max)
+AS
+BEGIN
+    IF NOT EXISTS(SELECT TOP 1 1 FROM Jurisdiction WHERE Name = @name)
+        BEGIN
+            INSERT INTO Jurisdiction (Code, Name, IsLive, CreatedDate, UpdatedDate) VALUES (@name, @name, 1, CURRENT_TIMESTAMP, CURRENT_TIMESTAMP)
+        END
+END
+GO
+
+CREATE OR ALTER PROC #CaseType_Update @name varchar(max), @serviceId nvarchar(max), @jurisdictionName nvarchar(max)
+AS
+BEGIN
+    DECLARE @jurisdictionId int = (SELECT Id FROM Jurisdiction WHERE Name = @jurisdictionName)
+
+    IF EXISTS(SELECT TOP 1 1 FROM CaseType WHERE Name = @name)
+        BEGIN
+            UPDATE CaseType SET ServiceId = @serviceId, JurisdictionId = @jurisdictionId, UpdatedDate = CURRENT_TIMESTAMP WHERE Name = @name
+        END
+    ELSE
+        BEGIN
+            Print ('WARNING! Could not find case type with the name: ' + @name);
```

Comment:
Did you find any warnings in the output? Since the ref data should be the same across all env I went back to the team to fill in any gaps

Reply:
it did help me find a case type specified in the ticket that didn't exist in the db, yes. I've since removed that case type from the script so it won't a throw a warning, but I've left the warning check in place as a safeguard just in case things change by the time we run this script

Reply:
nice. always useful to sense check this in dev but then capture problems in later environments